project_template/forms.py

```diff
@@ -169,6 +169,41 @@ class TranscribeOwnedIncomeFromDeferredUseOfGoods(forms.Form):
     count = forms.IntegerField(label="Câte rânduri completate există în tabelul {}?".format(constants.DECLARATION_TABLES['deferred_use']))
 
 
+class TranscribeOwnedIncomeFromDeferredUseOfGoodsRowEntry(forms.Form):
+    holder_relationship = forms.ChoiceField(label="Cine a realizat venitul?", choices=HolderRelationship.return_as_iterable())
+    holder_surname = forms.CharField(label="Care este numele persoanei care a realizat venitul?")
+    holder_name = forms.CharField(label="Care este prenumele persoanei care a realizat venitul?")
+    source_of_goods_source_surname = forms.CharField(label="Care este prenumele sursei de venit?", required=False)
```

Comment:
`source_of_goods_source_surname` & `source_of_goods_source_name` are redundant. 

To clarify: a Person (political figure) has one or more Declarations. A Declaration has one or more tables. A table has one or more rows. The name & surname that a volunteer transcribes from a row do not belong to the Person (the political figure who filled out the asset declaration). They belong to the "source of income / goods". 

Please remove these two fields.